tests/foreman/ui/test_settings.py

```diff
@@ -499,8 +481,9 @@ def test_negative_update_hostname_with_empty_fact(session, setting_update):
     new_hostname = ""
     property_name = setting_update.name
     with session:
-        response = session.settings.update(property_name, new_hostname)
-        assert response is not None, "Value can't be blank"
+        with pytest.raises(AssertionError) as context:
```

Comment:
Are you sure about `AssertionError` here? I think it should be an error that `session.settings.update` will throw, could be an HTTP Error !

Reply:
I think its appropriate, the `update` method hits `assert_no_errors`, and is specifically expecting that behavior here, not for any exception type to occur.

https://github.com/SatelliteQE/airgun/blob/0ad943f3d4149f85ecf9533bc6965ba0e4e18aca/airgun/widgets.py#L780

If the exception is an HTTPError (not AssertionError), something has definitely failed, and the test will end in ERROR because of the unhandled exception.

In the case of the AssertionError, it should contain the text of the message in the UI, which is then validated against.